### PR TITLE
test: E2E NDV

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: cypress/pages/workflow.ts [hunk 1]
```diff
@@ -11,17 +11,11 @@ export class WorkflowPage extends BasePage {
...
 		canvasPlusButton: () => cy.getByTestId('canvas-plus-button'),
-		canvasNodeBox: (nodeDisplayName: string) => {
-			return cy
-				.getByTestId('canvas-node-box-title')
-				.contains(nodeDisplayName)
-				.parents('[data-test-id="canvas-node-box"]');
-		},
-
+		canvasNodes: () => cy.getByTestId('canvas-node'),
+		canvasNodeByName: (nodeName: string) => this.getters.canvasNodes().filter(`:contains("${nodeName}")`),
```
Comment (OlegIvaniv) [inline cypress/pages/workflow.ts:18]: That's a nice trick to yield the root node while filtering for content. Much cleaner than cy.get(..).contains(..).parents(..). Will use it from now on :D

File: cypress/e2e/5-ndv.cy.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		workflowPage.actions.renameWorkflow(uuid());
+		workflowPage.actions.saveWorkflowOnButtonClick();
+	});
+
+	it('should show up when double clicked on a node and close when Back to canvas clicked', () => {
+		workflowPage.actions.addInitialNodeToCanvas('Manual Trigger');
+		workflowPage.getters.canvasNodes().first().dblclick();
+		cy.getByTestId('ndv').should('be.visible');
+		cy.getByTestId('back-to-canvas').click()
+		cy.getByTestId('ndv').should('not.be.visible');
```
Comment (OlegIvaniv) [inline cypress/e2e/5-ndv.cy.ts:21]: Why not create NDV getters for these? We might want to reuse them beyond this suite and it's much easier to get them suggested by IDE than searching for the right ids

Comment (cstuncsik) [reply cypress/e2e/5-ndv.cy.ts]: created a page object

**Review by OlegIvaniv — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.205.0`

**Pull request merged**